packages/lodestar/src/eth1/eth1MergeTransitionBlockTracker.ts

```diff
@@ -0,0 +1,349 @@
+import {AbortSignal} from "@chainsafe/abort-controller";
```

Comment:
Why is this file rendering as it's being added in this PR?

Reply:
no no, it will be removed, got some renaming mixed up. :slightly_smiling_face: